packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -111,7 +98,7 @@ export const getStyles = memoizeFunction((
       color: checkmarkFontColorCheckedDisabled,
     },
     text: {
-      margin: '0 4px'
+      marginRight: '8px'

```

Comment:
The styling package automatically handles RTL. If it detects that the page is in RTL, it will reverse any directional styles, including margins.